keps/sig-node/0034-20190130-node-os-arch-labels.md

```diff
@@ -0,0 +1,92 @@
+---
+kep-number: 34
+title: 
+authors:
+  - "@yujuhong"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+reviewers:
+  - "@liggitt"
+approvers:
+  - "@dchen1107"
+editor: Yu-Ju Hong
+creation-date: 2019-01-30
+last-updated: 2019-01-30
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Promote Node Operating System & Archtecture labels to GA
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+
+## Summary
+
+The kubelet has been been labeling the Node object with operating system (OS)
+and archietcture (arch) labels since Kubernetes 1.3. This proposal aims to
+promote these labels to GA and ensure a smooth transition with backward
+compatibility.
+
+Below lists the labels to promote:
+```
+beta.kubernetes.io/os
+beta.kubernetes.io/arch
+```
+
+## Motivation
+
+The labels have existed over two years and are widely in various places.
+Promoting these labels to reflect their stable status.
+
+### Goals
+
+Promote the OS and arch labels to GA.
+
+### Non-Goals
+
+Promoting any other label that is not listed in the Goals sections.
+
+## Proposal
+
+A new set of GA labels will be added:
+
+```
+kubernetes.io/os
+kubernetes.io/arch
+```
+
+Starting in *v1.14*,
```

Comment:
add:
- announce the deprecation of the beta OS and GA labels, and the timeframe for removal (v1.18)
- update documentation and examples to the GA labels (noting which version the GA labels were added)

Reply:
Done.